normative/dtpr-stickers
# Adds icons placeholders and place QRCode print

# Notion card link(s)
- [Add placeholder to missing sensor and taxonomy icons](https://www.notion.so/normative/Add-placeholder-to-missing-sensor-and-taxonomy-icons-9f01db6aa0fe4382994bd5c7355f5097)

# Review APP
- [Heroku Review App](https://dtpr-sticker-feature-ic-spnfdx.herokuapp.com/sensors/myd2vKdzrsn7RaF4t_UIb)

# PR Description (Proposed changes, approach, gotchas, etc)
- [x] Adds icons placeholders
- [x] Adds place QRCode url link to be printed

# How to test it
- Access Review App through the link on the top of description and check feedback questions
- Check if icons are ok, according [Figma](https://www.figma.com/file/t1ZIjYmejw3bNc9hxMNzGB/Helpful-Places---Working-File?node-id=114%3A3482)
- Check if QRCode is show correctly with link
